README.md

```diff
@@ -111,7 +111,7 @@ TIME     EVENT COMM             PID     PPID    FILENAME/EXIT CODE
 ## uprobe
 
 `uprobe` is an example of dealing with user-space entry and exit (return) probes,
-`uprobe` and `uretprobe` in libbpf lingo. It attached `uprobe` and `uretprobe`
+`uprobe` and `uretprobe`, in libbpf lingo. It attached `uprobe` and `uretprobe`
```

Comment:
let's also fix attached -> attaches typo, while at it?

Also, for Signed-off-by in commit message, you are supposed to use properly capitalized real name, if that's not a problem. Thanks.

Reply:
fixed.